app/controllers/submit.js

```diff
@@ -261,6 +268,10 @@ export default Ember.Controller.extend(BasicsValidations, NodeActionsMixin, Tagg
             node.save()
                 .then(() => this.send('next', this.get('_names.2')))
                 .catch(()=> this.send('error', 'Could not save information; please try again'));
+            //Hack here because run.next and run.scheduleOnce afterRender dont pick up until the next edit.
+            Ember.run.later(this, function() {
+                MathJax.Hub.Queue(['Typeset', MathJax.Hub, Ember.$('#abstract')[0]]);  // jshint ignore:line
+            }, 300);
```

Comment:
Also, any idea why these things don't pick up until after next edit? Just handwaving here, but is there for example some aspect of rendering that is affected by the save operation in line 268, and we're just waiting for the promise above to resolve?